samples/nrf9160/nrf_cloud_mqtt_multi_service/sample.yaml

```diff
@@ -1,10 +1,47 @@
 sample:
   name: nRF Cloud MQTT Multi Service Sample
+common:
+  platform_allow: nrf9160dk_nrf9160_ns thingy91_nrf9160_ns
+  build_only: true
+  build_on_all: true
+  tags: ci_build
 tests:
-  samples.nrf9160.nrf_cloud_mqtt_multi_service:
-    build_only: true
-    platform_allow: nrf9160dk_nrf9160_ns thingy91_nrf9160_ns
-    integration_platforms:
```

Comment:
> Have you looked into using cmock + unity for mocks and unit testing? This is commonly used in other unit tests, examples: ncs/nrf/applications/asset_tracker_v2/tests ncs/nrf/tests/lib/modem_trace ncs/nrf/tests/lib/modem_jwt There is also mocking support in ztest, but it's quite limited compared to cmock: https://docs.zephyrproject.org/latest/develop/test/ztest.html?highlight=ztest#mocking

We were hearing from others that use of cmock/unity was being discouraged because they don't run on Windows.

Reply:
@plskeggs  We are making efforts to continue using using cmock/unity. The need for having a strong mocking framework is clearly seen in this PR and will be used as a supporting argument to keep using cmock/unity. The dependency of cmock/unity on ruby is the only thing some of us have issues with. We are working on finding a good solution to it and will reach a consensus in a couple of days. Until then, I dont think anyone is discouraged from writing cmock/unity based unit tests.

Reply:
This should be a topic of discussion next week.  @glarsennordic is travelling now, and I will be starting tomorrow.